sox.features.js

```diff
@@ -1003,7 +1003,15 @@
 
       const NEWQUESTIONS = 'metaNewQuestionAlert-lastQuestions';
       const favicon = sox.site.icon;
-      const metaName = 'meta.' + sox.site.currentApiParameter;
+      const sitename = sox.site.currentApiParameter;
+
+      // If it is a special site, add 'meta' before it, if not, replace the 'stackexchange' from the URL with 'meta'.
+      if (sitename.match(/stackoverflow|superuser|serverfault|askubuntu|stackapps|mathoverflow/)) {
+        var metaName = 'meta.' + sitename;
+      } else {
+        metaName = sitename.replace('stackexchange','meta');
```

Comment:
Should this be `sitename.replace('stackexchange', 'meta.stackexchange')` instead?

Reply:
After making some tests, I realized that API will accept (for meta site of Operations Research) both `or.meta` and `or.meta.stackexchange` as `site`.

Reply:
Ah cool! This seems good then, will merge :)